[2025-09-07T18:18:22.412+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-09-07T18:18:22.470+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: train_model_dag_batch_iterative_85000.train_model_iterative manual__2025-09-07T18:15:25.922064+00:00 [queued]>
[2025-09-07T18:18:22.503+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: train_model_dag_batch_iterative_85000.train_model_iterative manual__2025-09-07T18:15:25.922064+00:00 [queued]>
[2025-09-07T18:18:22.504+0000] {taskinstance.py:2306} INFO - Starting attempt 2 of 2
[2025-09-07T18:18:22.550+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): train_model_iterative> on 2025-09-07 18:15:25.922064+00:00
[2025-09-07T18:18:22.567+0000] {standard_task_runner.py:63} INFO - Started process 509 to run task
[2025-09-07T18:18:22.577+0000] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'train_model_dag_batch_iterative_85000', 'train_model_iterative', 'manual__2025-09-07T18:15:25.922064+00:00', '--job-id', '322', '--raw', '--subdir', 'DAGS_FOLDER/train_dag.py', '--cfg-path', '/tmp/tmpm3xlhllv']
[2025-09-07T18:18:22.584+0000] {standard_task_runner.py:91} INFO - Job 322: Subtask train_model_iterative
[2025-09-07T18:18:22.807+0000] {task_command.py:426} INFO - Running <TaskInstance: train_model_dag_batch_iterative_85000.train_model_iterative manual__2025-09-07T18:15:25.922064+00:00 [running]> on host 31d1c5cf642c
[2025-09-07T18:18:23.045+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='train_model_dag_batch_iterative_85000' AIRFLOW_CTX_TASK_ID='train_model_iterative' AIRFLOW_CTX_EXECUTION_DATE='2025-09-07T18:15:25.922064+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-09-07T18:15:25.922064+00:00'
[2025-09-07T18:18:23.047+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-09-07T18:18:35.152+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-09-07T18:18:35.153+0000] {taskinstance.py:2905} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/baseoperator.py", line 401, in wrapper
    return func(self, *args, **kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 235, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 252, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/train_dag.py", line 99, in send_batches_to_train_api
    raise Exception(f"Batch {idx} failed: {response.text}")
Exception: Batch 1 failed: {"detail":"Training failed: Image feature batch failed: Internal Server Error"}
[2025-09-07T18:18:35.183+0000] {taskinstance.py:1206} INFO - Marking task as FAILED. dag_id=train_model_dag_batch_iterative_85000, task_id=train_model_iterative, run_id=manual__2025-09-07T18:15:25.922064+00:00, execution_date=20250907T181525, start_date=20250907T181822, end_date=20250907T181835
[2025-09-07T18:18:35.205+0000] {standard_task_runner.py:110} ERROR - Failed to execute job 322 for task train_model_iterative (Batch 1 failed: {"detail":"Training failed: Image feature batch failed: Internal Server Error"}; 509)
[2025-09-07T18:18:35.270+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 1
[2025-09-07T18:18:35.320+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-09-07T18:18:35.330+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
